### PR TITLE
Support targetDirectoryPrefix for main extension

### LINKED ISSUE #78: Target folder prefix in configuration
plugin configurations usually define target folder prefix. Implement if needed

### PR DESCRIPTION
**Pull request by eirnym**
closes #78

### CODE REVIEW

**Comment by hannesa2** [comment]
If this https://github.com/jsonschema2dataclass/js2d-gradle/pull/81 would work, you could demonstrate usage immediate in pull request

**Comment by hannesa2** [comment]
> If this #81 would work, you could demonstrate usage immediate in pull request

Now you could do this

**Comment by eirnym** [comment]
@hannesa2 added an example for android

File: demo/androidDev/app/build.gradle [hunk 1]
```diff
@@ -65,6 +65,7 @@ jsonSchema2Pojo {
     //source = files("${sourceSets.main.output.resourcesDir}/json")
     source.setFrom files("${project.rootDir}/schema")
 
+    targetDirectoryPrefix = 'generated/source/jsonschema2dataclass'
```
Comment (hannesa2) [inline demo/androidDev/app/build.gradle:68]: Now I understand what you mean. It's the path where to put the generated classes !
Is it the same/similar like `targetDirectory` ?
If yes, why call it xxx**Prefix** ?

Comment (eirnym) [reply demo/androidDev/app/build.gradle:68]: Each execution adds its index to the suffix and then each variant (for Android builds) adds its own suffix as well

Comment (eirnym) [reply demo/androidDev/app/build.gradle:68]: Old targetDirectory parameter worked the same way. This name clarified the usage

**Review by hannesa2 — APPROVED** [review]

**Pull request merged**